README.md

```diff
@@ -45,13 +45,19 @@ function render(string $markdown): string {
 }
 ```
 
-For complete compatibility with GitHub Flavored Markdown, support for embedded HTML must be enabled; it is disabled
-by default as a security precaution.
+FBMarkdown currently supports three types of Markdown sources, with plans to expand: trusted, sponsored, and user-generated content.
 
-```Hack
-$ctx = (new Markdown\ParserContext())->enableHTML_UNSAFE();
-$ast = Markdown\parse($ctx, $markdown);
-```
+- __Trusted content mode__: For the parser, (a) embedded HTML is enabled, and (b) all URI schemes are enabled and will be parsed as links. For the renderer, `<img` tags are processed normally. In other modes, the `<img` tag is by default part of the HTML tags denylist, which we filter out. 
```

Comment:
It seems strange to be describing other modes under this heading; just to check: is everything enabled/unfiltered in this mode? If not, we should probalby change that...

Reply:
Yeah everything is enabled. I guess the only weird thing is that the blacklist for HTML tags can only be modified to remove the `<img` tag. But I guess the GFM spec uses the `tagfilter` extension, so this behavior is expected?